beta/scripts/headingIDHelpers/validateHeadingIDs.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ */
+const fs = require('fs');
+const walk = require('./walk');
+
+/**
+ * Validate if there is a custom heading id and exit if there isn't a heading
+ * @param {string} line
+ * @returns
+ */
+function validateHeaderId(line) {
+  if (!line.startsWith('#')) {
+    return;
+  }
+
+  const match = /\{\/\*(.*?)\*\/}/.exec(line);
+  const id = match;
+  if (!id) {
+    console.error(
+      'Uh Oh seems like the markdown misses headings. Can you run yarn fix-headings to generate headings.'
```

Comment:
Let's just do "Run yarn fix-headings to generate headings." :)

Reply:
Done